biopet/sampleconfig.wdl

```diff
@@ -77,3 +79,41 @@ task SampleConfigCromwellArrays {
         memory: ceil(memory * memoryMultiplier)
     }
 }
+
+task CaseControl {
+    input {
+        File? toolJar
+        String? preCommand
+        Array[File]+ inputFiles
+        Array[File]+ sampleConfigs
+        String outputPath
+        String controlTag = "control"
+
+        Int memory = 4
+        Float memoryMultiplier = 1.5
+    }
+
+    String toolCommand = if defined(toolJar)
+        then "java -Xmx" + memory + "G -jar " + toolJar
+        else "biopet-sampleconfig -Xmx" + memory + "G"
+
+    command {
+        set -e -o pipefail
+        ~{preCommand}
+        mkdir -p $(dirname ~{outputPath})
+        ~{toolCommand} CromwellArrays \
+        -i ~{sep="-i " inputFiles} \
+        -s ~{sep="-s " inputFiles} \
+        ~{"-o " + outputPath} \
+        ~{"--controlTag " + controlTag}
+    }
+
+    output {
+        File outputFile = outputPath
+        Array[CaseControl] controls = read_json(outputFile)
```

Comment:
Better name it caseControls instead of controls?

Reply:
fixed